### PR TITLE
EN-20157 Change document type to use json (from jsonb)

### PR DESCRIPTION
**Pull request by chitang**

### CODE REVIEW

File: coordinatorlib/src/main/scala/com/socrata/datacoordinator/common/soql/sqlreps/NonLookupableRep.scala [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package com.socrata.datacoordinator.common.soql.sqlreps
+
+import java.sql.PreparedStatement
+
+import com.socrata.soql.types.SoQLValue
+
+trait NonLookupableRep {
```
Comment (aerust) [inline coordinatorlib/src/main/scala/com/socrata/datacoordinator/common/soql/sqlreps/NonLookupableRep.scala]: what enforces this won't be called?

Comment (chitang) [reply coordinatorlib/src/main/scala/com/socrata/datacoordinator/common/soql/sqlreps/NonLookupableRep.scala]: They will be called when we set a column as custom key.  We don't allow document type to be custom key.  json type in pg doesn't even support eq (jsonb does).

Comment (rjmac) [reply coordinatorlib/src/main/scala/com/socrata/datacoordinator/common/soql/sqlreps/NonLookupableRep.scala]: That should be done by making SoQLDocument not a PKableRep

**Comment by chitang** [comment]
We are hoping that we will upgrade postgres to 9.4+ until this is really needed.

**Pull request closed without merging**